bundles/CoreBundle/Command/Document/GeneratePagePreviews.php

```diff
@@ -43,9 +44,60 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Prefix for the document path, eg. https://example.com, if not specified, Pimcore will try use the main domain from system settings.',
                 null
+            )
+            ->addOption(
+                'parent',
+                'p',
+                InputOption::VALUE_OPTIONAL,
+                'Only create preview of documents in parent hierarchy (ID|path).'
+            )
+            ->addOption(
+                'exclude-patterns',
+                'e',
```

Comment:
we cannot use option "e" as it is already being used for environment variable 
![image](https://user-images.githubusercontent.com/5137917/135095858-5cf0ac9c-bcfe-4619-b68c-680196cc88a9.png)
```suggestion
                'x',
```